locale/de_DE/locale.xml

```diff
@@ -9,34 +9,13 @@
-	<message key="plugins.generic.markup.archive.portalTitle">PKP Open Journal Systems Dokumenten-Markup-Plug-In</message>
-	<message key="plugins.generic.markup.archive.processing">Dokumenten-Markup-Server. Status: IN BEARBEITUNG...</message>
-	<message key="plugins.generic.markup.archive.status">Dokumenten-Markup-Server antwortet: {$message}</message>
-	<message key="plugins.generic.markup.archive.supp_file_missing">Obwohl ein Eintrag einer "Document Markup Files"-Zusatzdatei existiert, fehlt seine Datei.</message>
-	<message key="plugins.generic.markup.archive.supp_missing">Der Artikel oder der Eintrag seiner "Document Markup Files"-Zusatzdatei fehlen. Versuchen Sie, den Artikel erneut hochzuladen, um die Markup-Dateien zu erzeugen.</message>
+	<message key="plugins.generic.markup.archive.noArticle">Kein solcher Artikel!</message>
+	<message key="plugins.generic.markup.archive.noArticleID">"Der Paramter der Artikel-ID ist ungültig oder fehlt."</message>
+	<message key="plugins.generic.markup.archive.noJournal">Anfrage erfordert eine Zeitschrift.</message>
 	<message key="plugins.generic.markup.description">Wenn ein/e Autor/in, Lektor/in oder Redakteur/in eine neue Version (im Format odt, doc oder docx) eines Artikels hochlädt, übermittelt dieses Modul sie an den in der Konfiguration genannten Dokumenten-Markup-Server und erhält eine gezippte Datei mit PDF-, XML-, EPUB- und HTML-Versionen dieses Artikels zusammen mit einem Literaturverzeichnis, das automatisch den Richtlinien der Zeitschrift formatiert ist. Diese gezippte Datei wird in dem Zusatzdateienordner aufbewahrt, damit sie im Begutachtungsprozess untersucht werden kann. Wenn der Artikel veröffentlicht werden kann, werden die PDF-, XML- und HTML-Fahnen ausgepackt und über das Inhaltsverzeichnis der Zeitschrift zugänglich gemacht.</message>
 	<message key="plugins.generic.markup.displayName">Dokumenten-Markup-Plug-In</message>
-	<message key="plugins.generic.markup.error">Fehler!</message>
-	<message key="plugins.generic.markup.errors.errorMessage">Kann keine XML- und HTML-Versionen von Artikeln holen. Das Dokumenten-Markup-Plug-In muss aktiviert sein, und der Zitationsstil muss in den Zeitschrifteneinstellungen angegeben werden.</message>
-	<message key="plugins.generic.markup.optional.cslStyleName">Der Name des Zitationsstils ist optional.</message>
-	<message key="plugins.generic.markup.optional.markupHostPass">Passwort ist optional.</message>
-	<message key="plugins.generic.markup.optional.markupHostUser">Benutzer/innenname ist optional.</message>
-	<message key="plugins.generic.markup.optional.reviewVersion">Schalter für die Begutachtungsversion ist optional.</message>
 	<message key="plugins.generic.markup.required.cslStyle">Angabe des Zitationsstils ist zwingend.</message>
 	<message key="plugins.generic.markup.required.markupHostURL">Angabe der Serveradresse ist zwingend.</message>
 	<message key="plugins.generic.markup.required.markupHostUser">Username is required.</message>
```

Comment:
It looks like there's quite a bit of English text in the German locale file; this probably isn't something you introduced, but generally speaking it's a good idea to remove anything like this, so that the translator will notice it's not complete when they next run through the translation process.

Reply:
https://github.com/kaschioudi/ojs3-markup/pull/53/commits/e2d0eb5b0fe438febff08fb088981851fea31c34